addons/studio-followers/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Add followers to your studios",
+  "description": "Creates a modal that you can add your followers to a studio.",
```

Comment:
I think 'Creates a modal that you can use to invite your followers to studios.' might work better both grammatically and logically.